tests/support/util.tcl

```diff
@@ -1154,8 +1154,7 @@ proc system_backtrace_supported {} {
 
     # libmusl does not support backtrace. Also return 0 on
     # static binaries (ldd exit code 1) where we can't detect libmusl
-    catch {
-        set ldd [exec ldd src/redis-server]
```

Comment:
In tcl, `return` in catch is used to exit the block of catch, it will leave catch and continue to execute 'return 0'.

```tcl
proc test {} {
    catch {
        puts "a"
        return
        puts "b"
    }
    puts "c"
}

test
```

output:
```
a
c
```